components/inputs/input-time.js

```diff
@@ -118,15 +131,19 @@ class InputTime extends LitElement {
 		this.labelHidden = false;
 		this.timeInterval = 'thirty';
 		this._dropdownId = getUniqueId();
-		this._formattedValue = formatTime(DEFAULT_VALUE);
 		this._timezone = formatTime(new Date(), {format: 'ZZZ'});
-		this._value = formatValue(DEFAULT_VALUE);
 	}
 
 	get value() { return this._value; }
 	set value(val) {
+		if (this.value === undefined) {
+			this._givenValue = val;
+			return;
+		}
+
 		const oldValue = this.value;
-		const time = parseValue(val);
+		const time = val === '' ? getDefaultTime(this.defaultValue) : parseValue(val);
```

Comment:
Is empty string the only case where we'd want to revert to the default? Like what happens if the value attribute is completely removed? i.e. I have this `<d2l-input-time value="09:00:00">` and then I go `elem.removeAttribute('value')` -- what gets passed into the setter here? I'm guessing `undefined`.

Reply:
It passes `null` apparently - I'll accommodate that.